src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/RecommendationControllerTest.kt

```diff
@@ -106,7 +105,8 @@ class RecommendationControllerTest() : IntegrationTestBase() {
     val result = repository.findByCrnAndStatus(crn, Status.DRAFT.name)
 
     assertThat(result[0].data.lastModifiedBy, equalTo("SOME_USER"))
-    assertThat(result[0].data.recommendation?.name, equalTo(Recommendation.FIXED_TERM.name))
+
+    assertThat(result[0].data.recallType?.value, equalTo("FIXED_TERM"))
```

Comment:
Aren't you already verifying this on line 96?

Reply:
removed